test/render.js

```diff
@@ -361,20 +361,21 @@ function writeTable(browserPlatformType) {
   output += '<tbody>';
   for (let row of features) {
     let feature = Object.keys(row).map((k) => row[k])[0];
-    let desc = `<code>${Object.keys(row)[0]}</code>`;
-    if (feature.mdn_url) {
-      desc = `<a href="${feature.mdn_url}"><code>${Object.keys(row)[0]}</code></a>`;
-    }
+    let desc = '';
+    if (feature.mdn_url) { desc += `<a href="${feature.mdn_url}">`; }
     if (feature.description) {
       let label = Object.keys(row)[0];
       // Basic support or unnested features need no prefixing
       if (label.indexOf('.') === -1) {
-        desc = feature.description;
+        desc += feature.description;
         // otherwise add a prefix so that we know where this belongs to (e.g. "parse: ISO 8601 format")
       } else {
-        desc = `<code>${label.slice(0, label.lastIndexOf('.'))}</code>: ${feature.description}`;
+        desc += `<code>${label.slice(0, label.lastIndexOf('.'))}</code>: ${feature.description}`;
       }
+    } else {
+      desc += `<code>${Object.keys(row)[0]}</code>`;
     }
+    if (feature.mdn_url) { desc += '</a>'; }
```

Comment:
I think it might be a bit cleaner, instead of having this and [line 365](https://github.com/mdn/browser-compat-data/pull/400/files#diff-524f61060e1d8147246f863be0a1463aR365), to build `desc` without the link, then wrap it at the end:

```
if (feature.mdn_url) {
  desc = `<a href="${feature.mdn_url}">${desc}</a>`;
}
```

Reply:
Good idea!